### ISSUE TITLE
excel-shared-runtime-scenario sample fails manifest validation

### ISSUE THREAD

**Issue by caolgood** [OfficeDev/PnP-OfficeAddins#80]
**URL of sample**

https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/excel-shared-runtime-scenario

**Describe the bug**

The provided manifest.xml validation script fails for this sample. So now I'm not sure how to correctly configure  a shared runtime with custom functions.

```
> office-toolbox validate -m manifest.xml

Calling validation service. This might take a moment...
-------------------------------------
Validation: Failed

Error #1: 
No Supported Office Products: There are no platforms which fulfil the requirements specified in the manifest. The most common reasons for this failure are issues in the Requirements section. Please review the manifest and try again. (link: undefined)
```

Removing the sharedRuntime requirement set fixes this. But am I really supposed to do that?

**To Reproduce**

1.  cd to sample
2. yarn install
3. yarn run validate

**Expected behavior**

Validate should pass

**Environment**
- Platform [PC desktop, Mac, iOS, Office Online]: Mac
- Host [Excel, Word, PowerPoint, etc.]: excel
- Office version number: n/a
- Operating System: Mac catalina
- Browser (if using Office Online): ______

**Comment by aafvstam** [comment]
**Issue**
The manifest has the Requirement Set specified:
`<Set Name="SharedRuntime" MinVersion="1.1"/>`

**Reason**
SharedRuntime is not part of the OfficeJS Requirement Sets here:
[Requirement Sets](https://github.com/OfficeDev/Office-Js-Requirement-Sets)

This was identified by @shaofengzhu and added a **PR** for this:
[Add SharedRuntime1.1 to Excel](https://github.com/OfficeDev/Office-Js-Requirement-Sets/pull/58)

**Fix**
Add **SharedRuntime**

@caolgood So you are right while it is not in the Requirement Sets your only option is to remove it for now.

@sumurthy Can we get this fixed? (and #75 please ;-) )

**Comment by shaofengzhu** [comment]
It was fixed.
